### PR TITLE
[QA] SISRP-21727 - Financial Resources - Emergency Loan Link with CS Link API

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-21727

master PR https://github.com/ets-berkeley-edu/calcentral/pull/5674

### CODE REVIEW

File: src/assets/templates/widgets/finances_links.html [hunk 1]
```diff
@@ -67,6 +67,17 @@ <h3 data-ng-bind="subcategory"></h3>
             <strong>View Form</strong>
           </a>
         </li>
+        <li data-ng-if="subcategory === 'Financial Assistance'">
+          <a
+            data-cc-campus-solutions-link-directive="emergencyLoanLink.url"
+            data-cc-campus-solutions-link-directive-enabled="{{emergencyLoanLink.isCsLink}}"
+            data-cc-campus-solutions-link-directive-text="emergencyLoanLink.backToText"
+            data-cc-campus-solutions-link-directive-cache="'finances'"
+            data-cc-outbound-enabled="emergencyLoanLink.show_new_window"
```
Comment (johncrossman) [inline src/assets/templates/widgets/finances_links.html]: @dfkaye I would expect `show_new_window` to be camelCase.

Comment (dfkaye) [inline src/assets/templates/widgets/finances_links.html]: @johncrossman You asked that it be snake_case at https://github.com/ets-berkeley-edu/calcentral/pull/5660#discussion-diff-71950529

File: src/assets/templates/widgets/finances_links.html [hunk 2]
```diff
@@ -67,6 +67,17 @@ <h3 data-ng-bind="subcategory"></h3>
...
+            data-ng-attr-title="{{emergencyLoanLink.description}}" data-ng-if="emergencyLoanLink.url">
```
Comment (johncrossman) [inline src/assets/templates/widgets/finances_links.html]: @dfkaye should this be `data-ng-if="emergencyLoanLink.description"`?

File: src/assets/javascripts/angular/controllers/widgets/financesLinksController.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ var _ = require('lodash');
 /**
  * Footer controller
  */
-angular.module('calcentral.controllers').controller('FinancesLinksController', function(apiService, campusLinksFactory, financesLinksFactory, $scope) {
+angular.module('calcentral.controllers').controller('FinancesLinksController', function(apiService, campusLinksFactory, financesLinksFactory, csLinkFactory, $scope) {
```
Comment (johncrossman) [inline src/assets/javascripts/angular/controllers/widgets/financesLinksController.js]: @dfkaye Fyi, we try to alphabetize the controller arg list.  If you amend this PR then please move `csLinkFactory` in arg order.

File: src/assets/templates/widgets/finances_links.html (hunk 1 above)
Comment (johncrossman) [inline src/assets/templates/widgets/finances_links.html]: @dfkaye Ha! Sorry 'bout that. My comment in #5660 was ill-advised; feeds consumed by the front-end js should be camelCase. Perhaps you can make this right in a future master-only PR.

File: src/assets/templates/widgets/finances_links.html (hunk 2 above)
Comment (dfkaye) [inline src/assets/templates/widgets/finances_links.html]: @johncrossman think it should be `emergencyLoanLink.url` as without that, there is no url for the href attribute.

File: src/assets/templates/widgets/finances_links.html (hunk 1 above)
Comment (dfkaye) [inline src/assets/templates/widgets/finances_links.html]: @johncrossman I'll put that in another master PR and then resubmit / redo the QA PR

**Comment by johncrossman** [comment]
Thanks @dfkaye  Merging. Follow-up fixes to code style need only go to master; the QA branch does not need a follow up PR in order to alphabetize args, etc.

**Pull request merged**